packages/styles/scss/patterns/blocks/content-group-pictograms/_content-group-pictograms.scss

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin content-group-pictograms {
+  .#{$prefix}--content-group-pictograms {
+    @include carbon--make-container;
+
+    &__row {
+      @include carbon--make-row;
+    }
+
+    &__col {
+      @include carbon--make-col-ready;
+      @include carbon--breakpoint('sm') {
```

Comment:
Again small doesn't do anything.

Reply:
this was removed on the latest commit, thanks!